laget.Azure.ServiceBus/Topic/TopicSender.cs

```diff
@@ -20,45 +23,58 @@ public interface ITopicSender
     public class TopicSender : ITopicSender
     {
         private readonly ITopicClient _client;
+        private readonly BlobContainerClient _blobContainerClient;
+
+        public TopicSender(string connectionString, TopicOptions options) :
+            this(new TopicClient(connectionString, options.TopicName, options.RetryPolicy), null)
+        { }
 
-        public TopicSender(string connectionString, TopicOptions options)
+        public TopicSender(string connectionString, TopicOptions options, string blobConnectionString, string blobContainer) :
```

Comment:
Vi brukar generellt sätta `: this(...)` och `: base(...)` på en ny rad, alltså att `:` ligger med på raden under.